src/openai/_client.py

```diff
@@ -157,7 +157,8 @@ def qs(self) -> Querystring:
     @override
     def auth_headers(self) -> dict[str, str]:
         api_key = self.api_key
-        return {"Authorization": f"Bearer {api_key}"}
+        headers = {} if api_key == "" else {"Authorization": f"Bearer {api_key}"}
```

Comment:
What is we set `api_key` to `None`?
It should also handle situation in which API key is not set through argument of env variable.